plugins/k8saudit-gke/rules/k8s_audit_rules_gke.yaml

```diff
@@ -0,0 +1,100 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2023 The Falco Authors.
+#
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+- required_engine_version: 15
+
+- required_plugin_versions:
+  - name: k8saudit-gke
+    version: 0.1.0
+  - name: json
+    version: 0.7.0
+
+# Allow the Google Container Engine robot serviceaccount
+- rule: Disallowed K8s User
+  exceptions:
+    - name: google-container-engine-robot
+      fields: [ka.user.name]
+      comps: [endswith]
+      values:
+        - ["@container-engine-robot.iam.gserviceaccount.com"]
+  override: 
+    exceptions: append
+
+# Additional images allowed to run as 'privileged' containers
+- list: falco_privileged_images
+  items: [
+    gke.gcr.io/netd,
+    gke.gcr.io/csi-node-driver-registrar,
+    gke.gcr.io/gcp-compute-persistent-disk-csi-driver,
+    gke.gcr.io/kube-proxy-amd64,
+    gke.gcr.io/k8s-dns-node-cache
+  ]
+  override:
+    items: append
```

Comment:
> Fri Mar  8 15:32:53 2024: Loading plugin 'k8saudit-gke' from file /usr/share/falco/plugins/libk8saudit-gke.so
> Fri Mar  8 15:32:53 2024: Loading plugin 'json' from file /usr/share/falco/plugins/libjson.so
> Fri Mar  8 15:32:53 2024: Validating rules file(s):
> Fri Mar  8 15:32:53 2024:    /__w/plugins/plugins/plugins/k8saudit-gke/rules/k8s_audit_rules.yaml
> Fri Mar  8 15:32:53 2024:    /__w/plugins/plugins/plugins/k8saudit-gke/rules/k8s_audit_rules_gke.yaml
> Error: /__w/plugins/plugins/plugins/k8saudit-gke/rules/k8s_audit_rules.yaml: Ok
> /__w/plugins/plugins/plugins/k8saudit-gke/rules/k8s_audit_rules_gke.yaml: Invalid
> 1 Errors:
> In rules content: (/__w/plugins/plugins/plugins/k8saudit-gke/rules/k8s_audit_rules_gke.yaml:0:0)
>     list 'falco_privileged_images': (/__w/plugins/plugins/plugins/k8saudit-gke/rules/k8s_audit_rules_gke.yaml:38:2)
> ------
> - list: falco_privileged_images
>   ^
> ------
> LOAD_ERR_VALIDATE (Error validating rule/macro/list/exception objects): List uses 'append' or 'override.items: append' but no list by that name already exists

The `falco_privileged_images` does not exist in `k8s_audit_rules.yaml` even it is referenced (i was not noticed because in this condition `ka.req.pod.containers.image.repository in (falco_privileged_images)` it is being parsed as a bare string). cc @LucaGuerra @Issif @jasondellaluce @alacuku 

@sboschman can you add an empty `falco_privileged_images` list in `k8s_audit_rules.yaml` ? This should fix the problem.

Reply:
the list is defined in the falco stable rules, which is bundled by default... (well except in ci it seems), so not sure adding an empty list in `k8s_audit_rules.yaml` is the correct way to go. Perhaps trying to add the stable rules to ci is more in line with what happens when users are trying to activate the k8saudit or k8saudit-gke plugins

Reply:
fixed with #431